projects/example-app/src/app/auth/services/auth.service.ts

```diff
@@ -14,7 +14,7 @@ export class AuthService {
      * Simulate a failed login to display the error
      * message for the login form.
      */
-    if (username !== 'test' && username !== 'ngrx') {
+    if (password !== 'test' && username !== 'ngrx') {
```

Comment:
The reason why we don't check for a password is to make the example app easier to run.
If we ask for a password, people need to know it to browse the application app.

Reply:
I got it,that's a good one

Reply:
I'm going to close this PR for now, we might revisit later.